packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/explorerKey.from.event..st

```diff
@@ -0,0 +1,9 @@
+key handling
+explorerKey: aChar from: view event: event
```

Comment:
Is there a reasons why this method has the term explorer in its name? Personally I think smth like `handleKey: aChar from: view event: event` would be more intuitive in this case.

Reply:
This seems to be the standard way of doing it in Squeak. We copied this method from `Object Explorer`. We agree with you, but also think that consistency is important.

Reply:
Yes, I’ve seen the method in `ObjectExplorer` as well, but that seems to be the only place where it occurs. And in that context, I think it makes sense as the class is called explorer

Reply:
Ok, thanks for checking. I will change the message name.